### PR TITLE
Feature/api catalogue web

### PR DESCRIPTION
**Pull request by Addips**
## What

Adding api-catalogue frontend project

## Why

To be able to test the frontend in the dev environment

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

**Review by sindrig — APPROVED** [review]

**Review by sindrig — CHANGES_REQUESTED** [review]
Can you add your team as CODEOWNER for this app please

**Review by dabbeg — CHANGES_REQUESTED** [review]
The setup and core files looks legit, I cannot review the other content though.
I can approve when you have added someone from your team to the CODEOWNERS file and they can review the content of your projects :slightly_smiling_face:

**Comment by Addips** [comment]
> Can you add your team as CODEOWNER for this app please

Should be done, added for all of the api-catalogue stuff :)

**Comment by arnifreyrs** [comment]
Please add a reviewer from the advania team

File: .github/CODEOWNERS
Comment (sindrig) [inline .github/CODEOWNERS:50]: should probably add newline here. i'll add it to the linting step eventually.

**Review by dabbeg — APPROVED** [review]
:+1:

**Review by saevar43 — DISMISSED** [review]
LGTM

**Review by saevar43 — APPROVED** [review]

**Review by saevarma — APPROVED** [review]
LGTM

**Review by sindrig — APPROVED** [review]
:tada:

**Pull request merged**
